assets/src/edit-story/app/rightClickMenu/test/useRightClickMenu.js

```diff
@@ -103,51 +125,16 @@ describe('useRightClickMenu', () => {
   });
 
   describe('Page selected from right click', () => {
-    it('should return menu items', () => {
+    it('should return the correct menu items', () => {
       const { result } = renderHook(() => useRightClickMenu(), {
         wrapper: RightClickMenuProvider,
       });
 
-      expect(result.current.menuItems).toStrictEqual([
-        {
-          label: 'Copy',
-          onClick: expect.any(Function),
-          onMouseDown: expect.any(Function),
-          shortcut: {
-            display: 'ctrl C',
-            title: 'Control C',
-          },
-        },
-        {
-          label: 'Paste',
-          onClick: expect.any(Function),
-          onMouseDown: expect.any(Function),
-          shortcut: {
-            display: 'ctrl V',
-            title: 'Control V',
-          },
-        },
-        {
-          label: 'Delete',
-          onClick: expect.any(Function),
-          onMouseDown: expect.any(Function),
-          shortcut: {
-            display: 'DEL',
-            title: 'Delete',
-          },
-        },
-        {
-          label: 'Duplicate Page',
-          onClick: expect.any(Function),
-          onMouseDown: expect.any(Function),
-          separator: 'top',
-        },
-        {
-          label: 'Delete Page',
-          onClick: expect.any(Function),
-          onMouseDown: expect.any(Function),
-          disabled: expect.any(Boolean),
-        },
+      const labels = result.current.menuItems.map((item) => item.label);
+      expect(labels).toStrictEqual([
+        ...expectedDefaultActions,
+        RIGHT_CLICK_MENU_LABELS.DUPLICATE_PAGE,
+        RIGHT_CLICK_MENU_LABELS.DELETE_PAGE,
       ]);
```

Comment:
Simplified this to just check the text of the buttons. The integration tests check the functionality